qualifier-jeremy.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import re
+import typing
+
+
+class ArticleField:
+    def __init__(self, field_type: typing.Type[typing.Any]):
+        self.field_type = field_type
+
+    def __set_name__(self, owner, name):
+        self.name = name
+
+    def __get__(self, obj, type=None) -> typing.Type[typing.Any]:
+        return obj.__dict__.get(self.name)
+
+    def __set__(self, obj, value) -> None:
+        if not isinstance(value, self.field_type):
```

Comment:
it's often better to not negate (see this sentence as a good example). Positive statements are better understandable
Having if-else-blocks would also improve the readibility here